x/crosschain/keeper/keeper_cross_chain_tx_vote_inbound_tx.go

```diff
@@ -70,6 +70,10 @@ func (k msgServer) VoteOnObservedInboundTx(goCtx context.Context, msg *types.Msg
 	if receiverChain == nil {
 		return nil, sdkerrors.Wrap(types.ErrUnsupportedChain, fmt.Sprintf("ChainID %d, Observation %s", msg.ReceiverChain, observationType.String()))
 	}
+	tss, found := k.GetTSS(ctx)
+	if !found {
+		ctx.Logger().Error("TSS not found")
```

Comment:
Shouldn't we return an error? We use `tss.TssPubkey` later which I believe would cause a panic

Reply:
We shouldn't return here, as deposit Tx does not need a TSS address. I added some conditional logic instead. Thanks for pointing this out